spec/custom_rules/ApiGatewayCacheEncryptedRule_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'spec_helper'
+require 'cfn-model'
+require 'cfn-nag/custom_rules/ApiGatewayCacheEncryptedRule'
+
+describe ApiGatewayCacheEncryptedRule do
+  context 'Api Gateway has cache encryption enabled' do
+    it 'returns offending logical resource id' do
```

Comment:
Minor recommended change to match test case functionality
```suggestion
    it 'returns no violating resources' do
```